.github/workflows/testing.yml

```diff
@@ -132,12 +133,13 @@ jobs:
       - name: Prepare test database
```

Comment:
Try `sudo systemctl start mysql` for starting mysql above.

Reply:
Please see https://github.com/wojsmol/db-command/pull/1/commits/fe680747bd6a159fad3e20fec122cf0c80d673c8 - without any other changes.